Tasks/AzureCLIV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -17,10 +17,10 @@
   "loc.input.help.args": "Arguments passed to the script",
   "loc.input.label.addSpnToEnvironment": "Access service principal details in script",
   "loc.input.help.addSpnToEnvironment": "Adds service principal id and key of the Azure endpoint you chose to the script's execution environment. You can use these variables: `$servicePrincipalId` and `$servicePrincipalKey` in your script",
+  "loc.input.label.useGlobalConfig": "Use global Azure CLI configuration",
+  "loc.input.help.useGlobalConfig": "If this is false, this task will use its own separate [Azure CLI configuration directory](https://docs.microsoft.com/en-us/cli/azure/azure-cli-configuration?view=azure-cli-latest#cli-configuration-file). This can be used to run Azure CLI tasks in *parallel* releases",
```

Comment:
One thing to note: for RC2 localization has already been done and hence this help text will keep showing in english only

Reply:
and label also